Home_Work_1/task_1.py

```diff
@@ -0,0 +1,14 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+number = int(input('Введите трёхзначное число: '))
+summ = 0  
+
+
+while (number > 0):
+    summ += int(number % 10)
+    number /= 10
```

Comment:
Нахождение остатка от деления не может возвращать дробное число, если в качестве делимого выступает тоже целое. Настоящее название этого оператора: остаток от **целочисленного** отделения.
```suggestion
    summ += number % 10
    number //= 10
```